PayKids/presentation/src/main/java/com/paykids/presentation/view/signIn/SignNicknameFragment.kt

```
@@ -0,0 +1,74 @@
+) : BaseFragment<FragmentSignNicknameBinding>() {
+    private val viewModel: SignViewModel by viewModels()
+
+    override fun initView() {
+        binding.etNick.filters = arrayOf(InputFilter.LengthFilter(14))
+    }
+
+    override fun initListener() {
+        super.initListener()
+
+        binding.etNick.addTextChangedListener(object : TextWatcher {
+            override fun beforeTextChanged(s: CharSequence?, start: Int, count: Int, after: Int) {}
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)